### PR TITLE
Edited BowerPHP tip

### PR DESCRIPTION
**Pull request by SecondeJK**
Just a quite proposed change to keep us up to date: BowerPHP is now in beta, not alpha.

### CODE REVIEW

File: cookbook/frontend/bower.rst [hunk 1]
```diff
@@ -25,8 +25,8 @@ it's installed correctly.
 .. tip::
 
     If you don't want to have NodeJS on your computer, you can also use
-    BowerPHP_ (an unofficial PHP port of Bower). Beware that this is still in
-    an alpha state. If you're using BowerPHP, use ``bowerphp`` instead of
+    BowerPHP_ (an unofficial PHP port of Bower). Be aware that this is currently
```
Comment (javiereguiluz) [inline cookbook/frontend/bower.rst]: I prefer to maintain `Beware` here instead of `Be aware`

**Comment by javiereguiluz** [comment]
👍 thanks @SecondeJK

File: cookbook/frontend/bower.rst (hunk 1 above)
Comment (SecondeJK) [inline cookbook/frontend/bower.rst]: Doesn't really make a difference to me - the version being the only important bit!

**Comment by xabbuh** [comment]
Thank you @SecondeJK.

**Pull request closed without merging**